source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1,5 +1,5 @@
 # A part of NonVisual Desktop Access (NVDA)
-# Copyright (C) 2006-2023 NV Access Limited, Babbage B.V., Cyrille Bougot
+# Copyright (C) 2006-2024 NV Access Limited, Babbage B.V., Cyrille Bougot, Sascha Cowley

```

Comment:
Thanks for updating the copyright, you're already covered with the reference to NV Access.

```suggestion
# Copyright (C) 2006-2024 NV Access Limited, Babbage B.V., Cyrille Bougot
```